staging/src/k8s.io/apiserver/pkg/util/flowcontrol/reqmgmt.go

```diff
@@ -165,16 +184,23 @@ type RequestDigest struct {
 
 func (reqMgmt *requestManagementSystem) Wait(requestDigest RequestDigest) (execute bool, afterExecute func()) {
 	for {
-		rmState := reqMgmt.curState.Load().(*RequestManagementState)
+		rmState := reqMgmt.curState.Load().(*requestManagementState)
+
+		var hashValue uint64
+		var priorityLevelName = reqMgmt.globalDefaultPriorityLevelConfigurationName
 		fs := rmState.pickFlowSchema(requestDigest)
-		// RequestManagementState is constructed to guarantee that the following succeeds
-		ps := rmState.priorityLevelStates[fs.Spec.PriorityLevelConfiguration.Name]
+		if fs != nil {
+			flowDistinguisher := requestDigest.ComputeFlowDistinguisher(fs.Spec.DistinguisherMethod)
+			hashValue = hashFlowID(fs.Name, flowDistinguisher)
+			priorityLevelName = fs.Spec.PriorityLevelConfiguration.Name
+		}
+
+		// requestManagementState is constructed to guarantee that the following succeeds
```

Comment:
I think not.  `digestConfigObjects` does not ensure that the new `requestManagementState` has the "workload-low" priority level.  And the config controller might invoke `digestConfigObjects` with a list of PriorityLevelConfiguration objects that does not include "workload-low".  The config controller calls `digestConfigObjects` once its informer syncs, and this is not synchronized with the PostStartHook that ensures config objects exist.

Reply:
I still do not see a guarantee that the "default" priority leel is in that map.